calcom/cal.com
# feat: if profile only has one public event-type, redirect to it

## What does this PR do?

- Redirect user directly to the event if only a single public event is present
- When clicked on the avatar in the event, navigate to the user profile

Fixes #12089

## Type of change




## How should this be tested?

- User must have a single public event type.
- Navigate to https://cal.com/user_name
- The page should be redirected to that public event
- When clicked on the avatar it should navigate to the user profile

## Mandatory Tasks
